frame/election-provider-multi-phase/src/lib.rs

```diff
@@ -595,16 +606,26 @@ pub mod pallet {
 			}
 		}
 
-		fn offchain_worker(n: T::BlockNumber) {
-			// We only run the OCW in the first block of the unsigned phase.
-			if Self::current_phase().is_unsigned_open_at(n) {
-				match Self::try_acquire_offchain_lock(n) {
-					Ok(_) => {
-						let outcome = Self::mine_check_and_submit().map_err(ElectionError::from);
-						log!(info, "miner exeuction done: {:?}", outcome);
+		fn offchain_worker(now: T::BlockNumber) {
+			let threshold = T::OffchainRepeat::get();
```

Comment:
Threshold should be enforced, thus I think it is better to read it directly in `try_acquire_offchain_lock`, rather than passing it in as arg, implying it is configurable

Reply:
exactly preventing something like https://github.com/paritytech/substrate/pull/8290/files#r594214298

Reply:
81ceca174